macOS/LocalPackages/NewTabPage/Sources/NewTabPage/ProtectionsReport/NewTabPageDataModel+ProtectionsReport.swift

```diff
@@ -23,6 +23,21 @@ extension NewTabPageDataModel {
     struct ProtectionsData: Encodable, Equatable {
         let totalCount: Int64
         let totalCookiePopUpsBlocked: Int64?
+
+        enum CodingKeys: String, CodingKey {
+            case totalCount, totalCookiePopUpsBlocked
+        }
+
+        func encode(to encoder: Encoder) throws {
+            var container = encoder.container(keyedBy: CodingKeys.self)
+            try container.encode(totalCount, forKey: .totalCount)
+
+            if let totalCookiePopUpsBlocked {
+                try container.encode(totalCookiePopUpsBlocked, forKey: .totalCookiePopUpsBlocked)
+            } else {
+                try container.encodeNil(forKey: .totalCookiePopUpsBlocked)
```

Comment:
Required manual encoding to add explicit encoding of `nil` (otherwise it is omitted by default).